mindmaps-graql/src/main/java/io/mindmaps/graql/internal/analytics/Analytics.java

```diff
@@ -264,4 +374,25 @@ public static String persistResource(String keySpace, Vertex vertex,
     public static String getVertexType(Vertex vertex) {
         return vertex.value(Schema.ConceptProperty.TYPE.name());
     }
+
+    private void checkResourceType(String type) {
```

Comment:
Be a bit more explicit in the method name what is it checking for (number in this case), or parameterise the method by the expected data types.